pkgs/data/fonts/googlefonts/default.nix

```
@@ -0,0 +1,200 @@
+{ stdenvNoCC
+, lib
```

```suggestion
{ 
  stdenvNoCC
, lib
```
This is preferred for better diffs.